packages/web-components/fast-foundation/src/utilities/composed-contains.ts

```diff
@@ -0,0 +1,26 @@
+import { composedParent } from "./composed-parent";
```

Comment:
No change now...but wondering if we should move both of these helpers to fast-element in the future. They seem pretty handy.

Reply:
Yea I think that or `fast-web-utilities` would make sense